package.json

```diff
@@ -20,14 +20,20 @@
     "clean": "rimraf commonjs dist es",
     "lint": "eslint {src,test-browser}/**/*.{js,ts,tsx}",
     "lint:fix": "npm run lint -- --fix",
+    "dev": "concurrently \"npm run build:commonjs:watch\" \"npm run build:umd:watch\" \"npm run build:es:watch\"",
     "test:browser": "karma start karma.conf.js",
     "test:browser:once": "npm run test:browser -- --single-run",
     "webpack:dev": "webpack --mode development --config webpack.config.js --progress --colors",
+    "webpack:dev:watch": "npm run webpack:dev -- --watch",
     "webpack:prod": "webpack --mode production --config webpack.config.js --progress --colors",
+    "webpack:prod:watch": "npm run webpack:prod -- --watch",
     "build": "npm run build:commonjs && npm run build:umd && npm run build:es",
-    "build:umd": "npm run webpack:dev && npm run webpack:prod",
+    "build:umd": "concurrently \"npm run webpack:dev\" \"npm run webpack:prod\"",
+    "build:umd:watch": "npm run webpack:dev:watch && npm run webpack:prod:watch",
     "build:commonjs": "tsc -m commonjs --outDir commonjs --p tsconfig.json",
+    "build:commonjs:watch": "npm run build:commonjs -- -w",
     "build:es": "tsc -m es2015 --outDir es --p tsconfig.json",
+    "build:es:watch": "npm run build:es -- -w",
```

Comment:
What are all these new options for? Seems to many of them, are they actually useful?

Reply:
I wanted to be able to run build in a watch mode since it is very useful when working on this SDK. All of those new build scripts are used inside of the `dev` script, I created them so that the `dev` script is more readable.